hail/python/hail/experimental/densify.py

```diff
@@ -0,0 +1,50 @@
+import hail as hl
+from hail.methods.misc import require_row_key_variant
+
+from hail import MatrixTable
```

Comment:
It is used by the documentation string.  @tpoterba, this is necessary, right?

Reply:
for this?
```
:class:`.MatrixTable`
```

No, sphinx doesn't look at the Python runtime for that.

Reply:
It doesn't?  How does it know where to link if there are duplicate symbols in different packages?

Reply:
the `.` searches upward and then downward. If you don't prefix with the `.`, then it will look only in the current module, as defined by the file structure.

Reply:
if there are duplicate symbols, then I don't know whether you get an error or a random selection.

Reply:
Great, thanks for clarifying!

Reply:
I removed the import.